src/app/pages/create-account/create-account.page.ts

```diff
@@ -20,31 +20,36 @@ export class CreateAccountPage implements OnInit {
   email: '';
 
   isSubmitted = false;
-
+  signupForm: FormGroup;
 
   constructor(private router: Router, private activatedRoute: ActivatedRoute, public formBuilder: FormBuilder, private toastCtrl: ToastController, private authService: AuthService) {
     if (this.router.getCurrentNavigation().extras.state){
           this.signupMethod = this.router.getCurrentNavigation().extras.state.data;
           this.mobile = this.router.getCurrentNavigation().extras.state.mobile;
           this.email = this.router.getCurrentNavigation().extras.state.email;
     }
-  }
 
-  signupForm = this.formBuilder.group({
-     "firstName" : ['', [Validators.required, Validators.minLength(2)]],
-     "lastName" : ['', [Validators.required, Validators.minLength(2)]],
-     "password" : ['', Validators.compose([
-       Validators.minLength(5),
-       Validators.required,
-       Validators.pattern('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])[a-zA-Z0-9]+$')
-     ])],
-     "confirmPassword" : ['', [Validators.required]],
-  }, {validator: PasswordValidator});
+    this.signupForm = this.formBuilder.group({
+         "firstName" : ['', [Validators.required, Validators.minLength(2)]],
+         "lastName" : ['', [Validators.required, Validators.minLength(2)]],
+         "password" : ['', Validators.compose([
+           Validators.minLength(5),
+           Validators.required,
+           Validators.pattern('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])[a-zA-Z0-9]+$')
+         ])],
+         "confirmPassword" : ['', [Validators.required]],
+      }, {validator: PasswordValidator});
+
+  }
 
   get errorControl() {
     return this.signupForm.controls;
   }
 
+  get formErrors() {
+    return this.signupForm.errors;
```

Comment:
Using these form group errors since we can't manually add errors to form controls. I even tried to create a dummy form control just for auth errors but the validation rules didn't seem to allow me to add custom errors on the fly. More research here is needed.